plutus-contract/src/Plutus/Contract/Test/ContractModel/Internal.hs

```diff
@@ -1497,6 +1497,7 @@ propRunActions_ actions =
 defaultCheckOptionsContractModel :: CheckOptions
 defaultCheckOptionsContractModel =
   defaultCheckOptions & emulatorConfig . initialChainState .~ (Left . Map.fromList $ zip knownWallets (repeat (Ada.lovelaceValueOf 100_000_000_000_000_000)))
+                      & allowBigTransactions
```

Comment:
Should we specify this somewhere (haddock comment or a note) that the tests using the Contract Model might be accurate when submitting to real Cardano network because of big transactions? Not sure where though.. What do you think?

Reply:
Thanks, after a second look this turned out not to be necessary.